README.md

```diff
@@ -44,13 +44,14 @@ PDFMtEd Editor is an easy-to-use graphical metadata editor that supports viewing
 
 **Screenshot**
 
-![screenshot of PDFMtEd Editor](https://github.com/Glutanimate/PDFMtEd/blob/master/screenshots/pdfmted-editor.png)
+![screenshot of PDFMtEd Editor](https://github.com/uriel1998/PDFMtEd/blob/7f96c7170e6fc66992635cb1bd94ba824ffa790d/screenshots/pdfmted-editor.png)
 
 **Features:**
 
 - easily process multiple files and folders right from your file manager
-- preview your documents in your default PDF viewer and file manager before editing them
+- preview your documents in your default PDF viewer before editing them
```

Comment:
Obviously, I'm not the maintainer but I don't think removing the browse button is a good idea.